test/integration/functional_test.go

```diff
@@ -493,7 +513,26 @@ func validateDryRun(ctx context.Context, t *testing.T, profile string) {
 	defer cancel()
 
 	// Too little memory!
-	startArgs := append([]string{"start", "-p", profile, "--dry-run", "--memory", "250MB", "--alsologtostderr"}, StartArgs()...)
+	var startArgs []string
+	orgStartArgs := StartArgs()
+
+	// remove '--memory=xxx' and '--memory xxx' arguments
```

Comment:
if this change is not related to this PR or not needed for arm64 lets not include it in this PR.

Reply:
done